pallets/registrar/src/lib.rs

```diff
@@ -443,34 +562,59 @@ pub mod pallet {
             Ok(())
         }
 
-        /// Pause container-chain from collating without removing its boot nodes nor its genesis config
+        /// Pause or unpause container-chain from collating. Does not remove its boot nodes nor its genesis config.
+        /// Only container-chains that have been marked as valid_for_collating can be paused.
         #[pallet::call_index(4)]
         #[pallet::weight(T::WeightInfo::pause_container_chain(T::MaxLengthParaIds::get()))]
-        pub fn pause_container_chain(origin: OriginFor<T>, para_id: ParaId) -> DispatchResult {
+        pub fn pause_container_chain(
+            origin: OriginFor<T>,
+            para_id: ParaId,
+            pause: bool,
```

Comment:
isnt this boolean implicit in the extrinsic?

Reply:
hmm I see if pause if false then it means unpause. I would add two extrinsics for this, I think its more desdcriptive and usable

Reply:
Good idea, I agree